code/controllers/Processes/mob.dm

```diff
@@ -1,27 +1,34 @@
 /datum/controller/process/mob
-	var/tmp/datum/updateQueue/updateQueueInstance
 
 /datum/controller/process/mob/setup()
 	name = "mob"
 	schedule_interval = 20 // every 2 seconds
-	updateQueueInstance = new
+	start_delay = 16
 	if(!mob_master)
 		mob_master = new
 		mob_master.Setup()
 
 /datum/controller/process/mob/started()
 	..()
-	if(!updateQueueInstance)
-		if(!mob_list)
-			mob_list = list()
-		else if(mob_list.len)
-			updateQueueInstance = new
+	if(!mob_list)
+		mob_list = list()
+
+/datum/controller/process/mob/statProcess()
+	..()
+	stat(null, "[mob_list.len] mobs")
 
 /datum/controller/process/mob/doWork()
-	if(updateQueueInstance)
-		updateQueueInstance.init(mob_list, "Life")
-		updateQueueInstance.Run()
-		mob_master.process()
+	for(var/mob/M in mob_list)
```

Comment:
I could make it behave like a queue instance by adding a sleep(0), but that's something that would belong in scheck... and, technically, _is_ in scheck, but will basically never happen, because it will defer first. I should probably look into simplifying scheck and making it do more "catch-up" sleeps.

A spawn(-1), on the other hand, will spawn off a thread that will run immediately, but can sleep without delaying further processing. The processing calls shouldn't sleep, however - if they do, it's a mistake, which needs to be detected and corrected (as I did for airlocks, which were blocking the machinery process for a whopping 6 seconds on game start). Note that update queues don't actually spawn off their processing calls, but rather, spawn recursive calls to their work function instead of simply sleeping in a loop. As far as I can tell, this serves no purpose but to generate extra spawn overhead.

You may have noticed that I use spawn(-1) in the bot process. I saw that something in the bot processing was sleeping (which is bad), peeked into the bot code, promptly noped on out of the bot code, draped a spawn over the processing call, and called it a day. It's not really the _right_ way to solve the problem, but it _is_ the way that didn't involve touching the bot code.